Makefile.am

```diff
@@ -193,6 +193,10 @@ doc:
 	@echo
 	@echo "Build finished. The HTML pages are in $(SPHINXBUILDDIR)/html."
 
+EXTRA_DIST += docs
```

Comment:
I'm not sure whether we want to pull in every file that lies under `docs/`. Despite we cleaned the generated html files there may be other files located in this directory in your working copy that will pollute the distribution.